### ISSUE TITLE
Python error in unittest (C++ error)

### ISSUE THREAD

**Issue by cyrenaique** [willyd/caffe-builder#4]
Hello,
I am facing a low level error in test_solver.py (test_net.py pass):
[libprotobuf ERROR D:\Libraries2\caffe_x3\build\DownloadCache\protobuf\src\google\protobuf\text_format.cc:275] Error parsing text-format caffe.SolverParameter: 1:11: Expected eight hex digits up to 10ffff for \U escape sequence

-OPENCV3
-Python3.4

Have you any clues about that one?
Thanks again.

**Comment by willyd** [comment]
Nope. All I can say is that I had trouble with protobuf because on Windows you absolutely need to specify it if you are reading a binary file unlike in Linux.   HTH.
